include/sce/types/kernel.h

```diff
@@ -2,11 +2,24 @@
 
 #include <stdint.h>
 #include <sys/time.h>
+#include <sys/event.h>
```

Comment:
If I just declare `struct knote` in my sample, this header give me to error due forwarding declaration of struct knote;
Also, ps4sdk-examples/posix/stdio-redirect need 1 line change to compile fine now;

Reply:
Hi @masterzorag! Thanks for the feedback and help! The example can be fixed by adding `#define __BSD_VISIBLE 1` before the includes.

But I would like to avoid including this header into the user-space as well. Instead of adding the include, can we use a `void pointer` to define `SceKernelEvent`? I see getter symbols for the fields of the structs, which suggests that this is the intended way of using the type. E.g. `sceKernelGetEventData: SceKernelEvent -> intptr_t`.

So, I would remove the `sys/event.h` include and use `typedef void * SceKernelEvent` instead. Then add the additional getters like `sceKernelGetEventData`

Reply:
@psxdev if you can fix the above issue I will merge the PR. Alternatively we can close it and you could add it to the libs directly (but there it should not be in /include/sce but in /include) so you will have to refactor your code. If that is ok we can close the PR - otherwise lets add the headers (after the fix above) so that for now you do not need to port your code to the new header locations.